src/modlunky2/ui/trackers/category.py

```diff
@@ -44,29 +43,43 @@ def __init__(self, parent, ml_config: Config, *args, **kwargs):
         self.category_button.grid(row=0, column=0, pady=5, padx=5, sticky="nswe")
 
         self.always_show_modifiers = tk.BooleanVar()
-        self.always_show_modifiers.set(False)
+        self.always_show_modifiers.set(
+            modlunky_config.trackers.category.always_show_modifiers
+        )
         self.always_show_modifiers_checkbox = ttk.Checkbutton(
             self,
             text="Always Show Modifiers",
             variable=self.always_show_modifiers,
             onvalue=True,
             offvalue=False,
+            command=self.toggle_always_show_modifiers,
         )
         self.always_show_modifiers_checkbox.grid(
             row=0, column=1, pady=5, padx=5, sticky="nw"
         )
 
+    def toggle_always_show_modifiers(self):
+        self.modlunky_config.trackers.category.always_show_modifiers = (
+            self.always_show_modifiers.get()
+        )
+        self.modlunky_config.save()
+        if self.window:
+            self.window.update_config(self.modlunky_config.trackers.category)
+
     def launch(self):
-        color_key = self.ml_config.tracker_color_key
+        color_key = self.modlunky_config.tracker_color_key
```

Comment:
Maybe not in this PR but would be nice to have some kind of migration strategy to move stuff like this out of the global space as we get more specific config sections.

Reply:
Yeah, I think some config migration infrastructure is needed